web/assets/js/model/outbound.js

```diff
@@ -897,4 +900,43 @@ Outbound.HttpSettings = class extends CommonClass {
             }],
         };
     }
+};
+Outbound.WireguardSettings = class extends CommonClass{
+    constructor(MTU, DNS, secretKey, address, publicKey, allowedIPs, endpoint) {
```

Comment:
It is also better to pre-define constructor values as much as possible, for example MTU and DNS

Reply:
Also use small letters for parameter's name: mtu, dns